dask/bytes/s3.py

```diff
@@ -33,15 +33,13 @@ def __init__(self, key=None, username=None, secret=None, password=None,
             secret = password
         if secret is not None:
             kwargs['secret'] = secret
-        self.blocksize = blocksize
```

Comment:
Since there was no way to change this value, whatever was passed in storage_options was indeed set in the S3FileSystem, but the open() here would always over-ride with the default value of blocksize to DaskS3FileSystem, which could not be changed.